client/ayon_core/addon/base.py

```diff
@@ -411,82 +411,56 @@ def _load_addons_in_core(
 ):
     # Add current directory at first place
     #   - has small differences in import logic
-    hosts_dir = os.path.join(AYON_CORE_ROOT, "hosts")
     modules_dir = os.path.join(AYON_CORE_ROOT, "modules")
+    if not os.path.exists(modules_dir):
+        log.warning((
+            "Could not find path when loading AYON addons \"{}\""
+        ).format(modules_dir))
+        return
 
-    for dirpath in {hosts_dir, modules_dir}:
-        if not os.path.exists(dirpath):
-            log.warning((
-                "Could not find path when loading AYON addons \"{}\""
-            ).format(dirpath))
-            continue
-
-        is_in_modules_dir = dirpath == modules_dir
-        ignored_filenames = set()
-        if is_in_modules_dir:
-            ignored_filenames = set(IGNORED_DEFAULT_FILENAMES)
-
-        for filename in os.listdir(dirpath):
-            # Ignore filenames
-            if filename in IGNORED_FILENAMES or filename in ignored_filenames:
-                continue
+    ignored_filenames = set(IGNORED_DEFAULT_FILENAMES)
 
-            fullpath = os.path.join(dirpath, filename)
-            basename, ext = os.path.splitext(filename)
+    for filename in os.listdir(modules_dir):
+        # Ignore filenames
+        if filename in IGNORED_FILENAMES or filename in ignored_filenames:
```

Comment:
If these are both sets might be worth merging it before the for loop to make this just:

```python
ignored_filenames = IGNORED_DEFAULT_FILENAMES + IGNORED_FILENAMES
if filename in ignored_filenames:
    continue
```

But maybe those two constants (now being constants?) could be merged on the module level already as well into one set.

Reply:
Cool - fine with me.